apps/cms-server/modules/openstad-auth/index.js

```diff
@@ -30,10 +32,39 @@ function removeURLParameter(url, parameter) {
   return url;
 };
 
+async function logout(req, res, next) {
+  // logout - ik kan het niet als functie aanroepen; daarom hier een kopie uit /node_modules/apostrophe/modules/@apostrophecms/login/index.js
+  const loggedInCookieName = 'loggedIn';
+  if (req.session) {
+    const destroySession = () => {
+      return require('util').promisify(function(callback) {
+        // Be thorough, nothing in the session potentially related to the login should survive logout
+        return req.session.destroy(callback);
+      })();
+    };
+    const cookie = req.session.cookie;
+    await destroySession();
+    // Session cookie expiration isn't automatic with `req.session.destroy`.
+    // Fix that to reduce challenges for those attempting to implement custom
+    // caching strategies at the edge
+    // https://github.com/expressjs/session/issues/241
+    const expireCookie = new expressSession.Cookie(cookie);
+    expireCookie.expires = new Date(0);
+    const name = self.apos.modules['@apostrophecms/express'].sessionOptions.name;
+    req.res.header('set-cookie', expireCookie.serialize(name, 'deleted'));
+
+    // TODO: get cookie name from config
```

Comment:
Moet dit daadwerkelijk nog gebeuren of is dit mee gekopieerd?

Reply:
Meegekopieerd inderdaad. En nu verwijderd...